<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditComputerDialog</class>
 <widget class="QDialog" name="EditComputerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="input_computer_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>378</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_computer_edit_cancel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>185</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="edit_computer_radiobutton_no">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>194</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_computer_wasItBuilt">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>171</y>
     <width>378</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Was it built</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_computer_year_built">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>114</y>
     <width>378</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_computer_date">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>137</y>
     <width>378</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_computer_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>23</y>
     <width>378</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_computer_edit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>185</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit Computer</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="edit_computer_radiobutton_yes">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>194</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_computer_type">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>57</y>
     <width>378</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_computer_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>378</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
